build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     implementation 'net.coobird:thumbnailator:0.4.19'
     implementation 'org.webjars.npm:dropzone:5.9.3'
     implementation 'com.amazonaws:aws-java-sdk-s3:1.12.364'
+    implementation 'org.apache.pdfbox:pdfbox:2.0.27'
```

Comment:
I'm noticing we've gotten very specific in our release numbers. Maybe this should be locked to major & minor but not patch?  Maybe this could be "2.0.+"?